build_3rdparty.py

```diff
@@ -88,8 +88,8 @@
     'libssh': {
         'order' : 3,
         'shadow': True,
-        'url'   : 'https://git.libssh.org/projects/libssh.git/snapshot/libssh-0.9.3.tar.xz',
-        'sha1'  : '7302fb1e4e91d3c7318b48276f30656688367e5a',
+        'url'   : 'https://www.libssh.org/files/0.9/libssh-0.9.3.tar.xz',
```

Comment:
@woa7 why did you change this? We had problems to build with this url from appveyor and therefore changed intentionally to the git snapshot